### PR TITLE
Create navbar for ScholarX

### LINKED ISSUE #78: Create a navbar for scholaraX
**Is your feature request related to a problem? Please describe.**
- We need to create a navbar for the scholarX platform

**Describe the solution you'd like**
- Create a layout component
- Create a navbar
   - The navbar contains the logo, scholarX logo and sign in sign out buttons.
 
scholarx

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

The purpose of this PR is to fix #78 

## Goals

- Create a navbar for scholarX site.

## Approach

- Add revelant code lines for ```/NavigationBar/index.tsx```
- Add styles to ```/NavigationBar/styles.css```
- Add the navigation bar to the relevant pages.

## Screenshots

Screenshot 2021-02-19 at 15 33 27

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
macOS BigSur, Google Chrome, Visual Studio Code.

### CODE REVIEW

File: src/scenes/Home/components/NavigationBar/styles.css [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+}
+
+.loginComponents {
+    position: relative;
+    float: right;
+}
+
+.navButtons {
+    color: grey;
+}
```
Comment (Gimhan-minion) [inline src/scenes/Home/components/NavigationBar/styles.css]: Add a new line here :)

**Review by Gimhan-minion — DISMISSED** [review]
This looks awesome @kumuditha-udayanga Good job!

**Review by Gimhan-minion — DISMISSED** [review]
Thank you @kumuditha-udayanga

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Good Job! @kumuditha-udayanga 
A minor change.
Can we have a margin like this?

108489670-d276ee00-72c7-11eb-82a4-aaa2df8054a6

**Comment by Gravewalker666** [comment]
Hello, @anjula-sack @kumuditha-udayanga, just to be clear if we need the navbar in every view just adding it to the root index file is sufficient, ain't it? Or are we only adding it to a few specific interfaces?

**Comment by kumuditha-udayanga** [comment]
Hi @Gravewalker666, as I found out it doesn't work, I tried opening other interfaces but it didn't work until I add them separately.

**Review by anjula-sack — APPROVED** [review]
Thank you! @kumuditha-udayanga

**Pull request merged**